packages/web3-eth-accounts/test/fixtures/account.ts

```diff
@@ -29,3 +31,186 @@ export const invalidPrivateKeytoAccountData: [any, string][] = [
 	[undefined, 'Invalid value given "undefined". Error: not a valid string or buffer.'],
 	[null, 'Invalid value given "null". Error: not a valid string or buffer.'],
 ];
+
+export const validEncryptData: [[any, string, CipherOptions], any][] = [
```

Comment:
`validEncryptData: [[any, string, CipherOptions], any][]`

should be:

`validEncryptData: [[HexString, string | Buffer, CipherOptions], Keystore][]`